test/tiles.spec.js

```diff
@@ -0,0 +1,171 @@
+    RED_FLAG: { id: 3, traversable: !myColorIsBlue },
+    RED_FLAG_TAKEN: { id: 3.1, traversable: true },
+    BLUE_FLAG: { id: 4, traversable: myColorIsBlue },
+    BLUE_FLAG_TAKEN: { id: 4.1, traversable: true },
+    SPEEDPAD_ACTIVE: { id: 5, traversable: false, radius: 15 },
+    SPEEDPAD_INACTIVE: { id: '5.1', traversable: true },
+    POWERUP_SUBGROUP: { id: 6, traversable: false, radius: 15 },
+    JUKEJUICE: { id: 6.1, traversable: false, radius: 15 },
+    ROLLING_BOMB: { id: 6.2, traversable: false, radius: 15 },
+    TAGPRO: { id: 6.3, traversable: false, radius: 15 },
+    MAX_SPEED: { id: 6.4, traversable: false, radius: 15 },
+    SPIKE: { id: 7, traversable: false, radius: 14 },
+    BUTTON: { id: 8, traversable: false, radius: 8 },
+    INACTIVE_GATE: { id: 9, traversable: true },
+    GREEN_GATE: { id: 9.1, traversable: false },
+    RED_GATE: { id: 9.2, traversable: !myColorIsBlue },
+    BLUE_GATE: { id: 9.3, traversable: myColorIsBlue },
+    BOMB: { id: 10, traversable: false, radius: 15 },
+    INACTIVE_BOMB: { id: '10.1', traversable: true },
+    RED_TEAMTILE: { id: 11, traversable: true },
+    BLUE_TEAMTILE: { id: 12, traversable: true },
+    ACTIVE_PORTAL: { id: 13, traversable: false, radius: 15 },
+    INACTIVE_PORTAL: { id: '13.1', traversable: true },
+    SPEEDPAD_RED_ACTIVE: { id: 14, traversable: false, radius: 15 },
+    SPEEDPAD_RED_INACTIVE: { id: 14.1, traversable: true },
+    SPEEDPAD_BLUE_ACTIVE: { id: 15, traversable: false, radius: 15 },
+    SPEEDPAD_BLUE_INACTIVE: { id: 15.1, traversable: true },
+    YELLOW_FLAG: { id: 16, traversable: true },
+    YELLOW_FLAG_TAKEN: { id: '16.1', traversable: true },
+    RED_ENDZONE: { id: 17, traversable: true },
+    BLUE_ENDZONE: { id: 18, traversable: true },
+    RED_BALL: { id: 'redball', traversable: true },
+    BLUE_BALL: { id: 'blueball', traversable: true },
+  };
+  const mockTileNames = {};
+  keys(mockTileInfo).forEach(key => {
+    mockTileNames[mockTileInfo[key].id] = key;
+  });
+  TileRewireAPI.__Rewire__('tileInfo', mockTileInfo);
+  TileRewireAPI.__Rewire__('tileNames', mockTileNames);
+}
+
+export function teardownTiles() {
+  TileRewireAPI.__ResetDependency__('tileInfo');
+  TileRewireAPI.__ResetDependency__('tileNames');
+}
+
+test('getTileProperty: returns correct properties', t => {
+  setupTiles(true);
+  t.is(getTileProperty(1, 'traversable'), false);
+  t.is(getTileProperty(2, 'traversable'), true);
+  t.is(getTileProperty(13, 'radius'), 15);
+  teardownTiles();
+  t.end();
+});
+
+test('getTileProperty: errors given tileIds that don\'t exist', t => {
+  setupTiles(true);
+  t.throws(() => { getTileProperty(1.123, 'traversable'); });
+  t.throws(() => { getTileProperty(-1, 'traversable'); });
+  t.throws(() => { getTileProperty('potato', 'traversable'); });
+  t.throws(() => { getTileProperty(undefined, 'traversable'); });
+  teardownTiles();
+  t.end();
+});
+
+test('isTraversable: correctly returns true for varying inputs', t => {
```

Comment:
Didn't we decided against the phrase `varying inputs` in test descriptions? (I would like to note that I still think it's useful phrasing.)

Maybe `orthogonal inputs` sounds less lazy?